Scala/Scala.sublime-syntax

```diff
@@ -249,16 +304,26 @@ contexts:
                 3: variable.import.renamed-to.scala
             - match: '([^\s.,}]+)'
               scope: variable.import.scala
+
   inheritance:
     - match: '(extends|with)\s+([^\s\{\(\[\]]+)'
       captures:
         1: keyword.declaration.scala
         2: entity.other.inherited-class.scala
+    - match: '(extends|with)\s+\('
```

Comment:
Then let’s add it to both! :smile:

In general I tried not to deviate into fixes that need to be made that were not touched by this PR (there are many related to parentheses and brackets, plus meta scopes). In this case, it is pretty minor, so we might as well include them both.